app/actions/ActionTypes.ts

```diff
@@ -245,6 +245,7 @@ export const NotificationsFeed = {
  */
 export const User = {
   FETCH: generateStatuses('User.FETCH') as AAT,
+  FETCH_LEADERBOARD: generateStatuses('User.FETCH') as AAT,
```

Comment:
Try to avoid using the same status for different actions

Reply:
I agree - this must be split up to its own `Achievements` action type